backend/de.metas.manufacturing/src/main/java/org/eevolution/costing/BatchProcessBOMCostCalculatorRepository.java

```diff
@@ -160,11 +165,15 @@ private BOM toCostingBOM(final I_PP_Product_BOM bomRecord)
 		final UomId uomId = UomId.ofRepoId(bomRecord.getC_UOM_ID());
 		final I_C_UOM uom = uomDAO.getById(uomId);
 
+		final CurrencyConversionContext conversionContext = currencyBL.createCurrencyConversionContext(SystemTime.asLocalDate(), (CurrencyConversionTypeId)null, clientId, orgId)
```

Comment:
pls take the date as param.

u could introduce BOMCostCalculator.evaluationDate
and u could set it from RollupBillOfMaterial